de/nmichael/efa/gui/EfaBaseFrame.java

```diff
@@ -103,19 +103,23 @@
     AdminRecord remoteAdmin;
     DataKeyIterator iterator;       // iterator for this logbook
     LogbookRecord currentRecord;    // aktDatensatz = aktuell angezeigter Datensatz
-    LogbookRecord referenceRecord;  // refDatensatz = Referenz-Datensatz (zuletzt angezeigter Datensatz, wenn neuer erstellt wird)
+    LogbookRecord referenceRecord;  // refDatensatz = Referenz-Datensatz (zuletzt angezeigter Datensatz, wenn neuer
+    // erstellt wird)
```

Comment:
@tfyh Too many formatting changes to review the actual changes. Please revert formatting changes.